src/components/Auth/Login/tests/LoginActions.test.js

```diff
@@ -0,0 +1,82 @@
+import reduxThunk from 'redux-thunk';
+import configurestore from 'redux-mock-store';
+import * as moxios from 'moxios';
+import {userLoginRequest} from '../../../../redux-js/actions/LoginAction';
+import {notify} from 'react-notify-toast';
+
+
+const middlewares = [reduxThunk];
+const mockStore = configurestore(middlewares);
+notify.show = jest.fn();
+const mockData = {
+	user: {
+		email: 'essa1@andela.com',
+		password: 'essa1@andela',
+
+	}
+};
+
+const errorMockData = {
+	errors: {"name": "is Required"}
+};
+
+describe('login component', () => {
+
+	const store = mockStore();
+	beforeEach(() => {
+		store.clearActions();
+		moxios.install();
+	});
+	afterEach(() => {
+		moxios.uninstall();
+	});
+
+	it('should login a user', () => {
+		moxios.stubRequest('https://ah-backend-staging.herokuapp.com/api/users/login/', {
```

Comment:
Please remove `URLS` from your code and define them in the constants.js file. Then you can have a single point of change.

Reply:
> Please remove `URLs` from your code and define them in the constants.js file. Then you can have a single point of change.

Thanks for pointing out. I have handled this.